e2e/favorites.spec.ts

```diff
@@ -101,7 +104,9 @@ test.describe('Favorites', () => {
         await expect(addFavorite).not.toBeVisible()
         await expect(removeFavorite).toBeVisible()
 
+        const spinner = getFavoriteStarSpiner(userPage)
         await removeFavorite.click()
+        await expect(spinner).not.toBeVisible()
```

Comment:
When an entity is favorited/unfavorited, a spinner appears until two calls (add/remove favorite, get favorites) resolve, then the favorites star button updates. Intermittently, the `expect` for the favorite star button update would timeout. Try waiting for the spinner to be removed, then check the favorites star button, so there is more time for the responses to finish.